src/web/components/SharingPermission/bulkAddPermissionsHelpers.ts

```diff
@@ -31,40 +30,63 @@ export const getDefaultDataProviderCheckboxState = (participantTypeNames: string
 export const getRecommendedTypeFromParticipant = (participantTypeNames: string[]) => {
   const uniqueTypes = new Set<string>();
   if (getDefaultPublisherCheckboxState(participantTypeNames)) {
-    uniqueTypes.add('Publishers');
+    uniqueTypes.add('publisher');
   }
   if (getDefaultAdvertiserCheckboxState(participantTypeNames)) {
-    uniqueTypes.add('Advertisers');
+    uniqueTypes.add('advertiser');
   }
   if (getDefaultDSPCheckboxState(participantTypeNames)) {
-    uniqueTypes.add('DSPs');
+    uniqueTypes.add('dsp');
   }
   if (getDefaultDataProviderCheckboxState(participantTypeNames)) {
-    uniqueTypes.add('Data Providers');
+    uniqueTypes.add('data_provider');
   }
-  const result = Array.from(uniqueTypes);
-  if (result.length === 4) return 'participants';
+  return Array.from(uniqueTypes);
+};
+
+const getParticipantTypeNamesMessage = (recommendedTypes: string[]) => {
+  if (recommendedTypes.length === 4) return 'participants';
+  const friendlyNames: Record<string, string> = {
```

Comment:
We've talked about finding a better solution for participant types, in this case we are mapping from the format we need to interact with admin with to the friendly plural for the message.